src/core/reducers/errors.js

```diff
@@ -11,19 +17,20 @@ import log from 'core/logger';
  * http://addons-server.readthedocs.io/en/latest/topics/api/overview.html#responses
  */
 function getMessagesFromError(error) {
-  let errorData = {
+  const errorData = {
     code: ERROR_UNKNOWN,
     messages: [],
   };
   log.info('Extracting messages from error object:', error);
 
+  const logCodeChange = ({ oldCode, newCode }) => {
+    log.warn(`Replacing error code ${oldCode} with ${newCode}`);
+  };
+
   if (error && error.response && error.response.data) {
+    // Extract a code and messages from the JSON response.
     Object.keys(error.response.data).forEach((key) => {
       const val = error.response.data[key];
```

Comment:
I know this isn't your fault, but can we change `val` to something more meaningful? At the very least, `value` 😄

Reply:
Sure